pandas/tests/io/test_excel.py

```diff
@@ -58,8 +51,9 @@ def ignore_xlrd_time_clock_warning():
         yield
 
 
-@td.skip_if_no('xlrd', '1.0.0')
-class ReadingTestsBase:
+@td.skip_if_no('xlrd')
```

Comment:
Note that our project-wide minimum ix 1.0.0, so this decorator was being duplicative.

This decorator also won't be there in the long term once we completely couple TestReaders from xlrd, but keeping here for now